mlflow/paddle/__init__.py

```diff
@@ -196,7 +195,10 @@ def forward(self, inputs):
     model_data_subpath = "model"
     output_path = os.path.join(path, model_data_subpath)
 
-    paddle.jit.save(pd_model, output_path)
+    if pd_model.__class__.__name__ == "Model":
```

Comment:
```suggestion
    if ininstance(pd_model, paddle.Model):
```

Can we use `isinstance` here?

Reply:
Changed! Thx so much!